testData/preferences/phil.json

```diff
@@ -0,0 +1,13 @@
+{
+    "flat": {
+        "contexts": {
+            "gpii-default": {
+                "name": "Default preferences",
+                "preferences": {
+                    "http://registry.gpii.net/common/slowKeysEnable": true,
```

Comment:
The first 3 settings should already be on.

For me, "Turn on Filter Keys" is ticked, and "Turn on Repeat Keys and Slow Keys" is on. Then, in the "Set up Repeat Keys and Slow Keys" section, the "Avoid accidental keystrokes" should be set to 0.5 seconds.

I've somehow managed to get both "Bounce key" and "Repeat/slow key" radio options to become enabled. This could be because I had Ease of Access Centre open, or because these settings are applied "in memory" rather than writing to the user's profile. So I think the best way to test these settings would be to test the effect (taste the coffee rather than check the ingredients).

But yes, this isn't an issue with this preference set.